crates/bevy_ecs/src/lib.rs

```diff
@@ -1530,6 +1531,7 @@ mod tests {
     #[test]
     fn filtered_query_access() {
         let mut world = World::new();
+        world.remove_resource::<DefaultQueryFilters>();
```

Comment:
I think it might be worth adding a comment explaining that the DefaultQueryFilters modify the access and would make the test harder to reason about